test/beacon/Voucher.test.ts

```diff
@@ -83,38 +137,22 @@ describe('Voucher', function () {
 
     describe('Upgrade', function () {
         it('Should upgrade all vouchers', async function () {
-            const { beacon, voucherHub, admin, voucherOwner1, voucherOwner2 } =
-                await loadFixture(deployFixture);
-            const voucherType1 = 1;
-            const duration1 = 7200;
-            const description1 = 'Long Term Duration';
-            const voucherValue1 = 200;
```

Comment:
The point was not to have a different duration of the common one ?

Reply:
Good catch!
[Create voucher with different type and value](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/18/commits/295037d0de82487ac453f31bced067556e48c21c)